utils/utils.mk

```diff
@@ -0,0 +1,26 @@
+#
+# Utility widgets for makefiles
+#
+# This file is intended to be included into other makefiles.  It provides useful
+# devices of various sorts which are applicable to a variety of situations.
```

Comment:
🏆 for defining "utility": "useful devices of various sorts [...] applicable to a variety of situations"? I love it! 🤣

Reply:
🙇